integration-tests/sql/src/main/java/org/apache/camel/quarkus/component/sql/it/SqlRoutes.java

```diff
@@ -114,7 +123,9 @@ private Path createTmpFileFrom(String file) throws IOException {
             while ((c = is.read()) >= 0) {
                 baos.write(c);
             }
-            fos.write(baos.toByteArray());
+            String content = new String(baos.toByteArray());
```

Comment:
Is it possible to remove this replaceAll if we use distinct files for projectsViaClasspath and projectsViaFile ?
Otherwise, would a comment make it clear that we reuse the same file in order load the sql request from file or from classpath ?

Reply:
I'll add it there - this code was present before current change, so I didn't see a need of commenting it. But I agree that it would make code easier to understand

Reply:
done